scripts/dockerfiles/Dockerfile.release

```diff
@@ -21,6 +21,7 @@ FROM amazonlinux:2
 RUN yum update -y && \
     yum install -y iptables && \
     yum install -y iproute && \
+    yum install -y nmap && \
```

Comment:
Instead of installing nmap package, probably, we check if gRPC is listening on the port by writing to it like below: 
`(echo >/dev/tcp/localhost/50051) &>/dev/null`

This is a lightweight alternative to using lsof/nc/netcat. wget/curl doesn't work here because it is a TCP connection, not an HTTP.

Reply:
Yes, I thought about that. The issue with that is we'd need to mount yet another host-path volume into this container.

Reply:
@mogren what's your thought on this one? prefer another host-path volume mount versus installing nmap?

Reply:
We could use the `app/grpc_health_probe` binary instead!